vm/src/obj/objbytes.rs

```diff
@@ -201,3 +202,15 @@ fn bytes_iter(obj: PyBytesRef, _vm: &VirtualMachine) -> PyIteratorValue {
         iterated_obj: obj.into_object(),
     }
 }
+
+impl PyBytesRef {
+    fn add(self, other: PyObjectRef, vm: &VirtualMachine) -> PyResult {
+        if objtype::isinstance(&other, &vm.ctx.bytes_type()) {
+            let rhs = get_value(&other);
+            let elements: Vec<u8> = self.value.iter().chain(rhs.iter()).cloned().collect();
+            Ok(vm.ctx.new_bytes(elements))
+        } else {
+            Err(vm.new_type_error(format!("Cannot add {} and {}", self.as_object(), other)))
```

Comment:
This should actually return `Ok(vm.ctx.not_implemented())` so that the vm can attempt the mirrored op (radd).

(A lot of our code is still getting this wrong.)

https://docs.python.org/3/library/constants.html#NotImplemented

Reply:
ok , we keep this kind of message ?

Reply:
If you return `NotImplemented` the vm will generate the message for you (if that's what you're asking)

Reply:
thats it